raml/schemas/definitions/group-output.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "roles": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "access": {
+            "type": "string"
+          },
+          "_id": {
+            "type": "string"
+          },
+          "site": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "access",
+          "_id"
```

Comment:
`roles` on groups and `permissions` on projects/sessions/acquisitions will all follow the same schema, they just have different field names (`roles` vs `permissions`). We'll probably want to add a schema snippet for that.